src/components/UnderlyingToken.tsx

```diff
@@ -21,7 +19,6 @@ type UnderlyingTokenProps = {
 export const UnderlyingToken = ({ assetId }: UnderlyingTokenProps) => {
```

Comment:
Curious if you have thought about this component leveraging zapper since it contains the underlying assets as well it looks like and then yearn can be removed entirely?

Reply:
Good point - we actually have `<EarnOpportunities />` that works with DeFi opportunities so I reckon we should remove this entirely along with `useYearn()`